staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks.go

```diff
@@ -457,8 +457,12 @@ type Disks interface {
 	// serialized as JSON into Description field.
 	CreateRegionalDisk(name string, diskType string, replicaZones sets.String, sizeGb int64, tags map[string]string) (*Disk, error)
 
-	// DeleteDisk deletes PD.
-	DeleteDisk(diskToDelete string) error
+	// DeleteDiskUnknownZone deletes the referenced persistent disk.
+	// Deprecated: prefer DeleteZonalDisk if the zone is known.
+	DeleteDiskUnknownZone(diskToDelete string) error
```

Comment:
Need to rename DeleteDisk to DeleteDiskUnknownZone in staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks_test.go